From 872748aa86d19067613784ea5f5cd1286d48793f Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Wed, 7 Dec 2022 10:04:46 +0100
Subject: [PATCH] ARM: dts: stm32: enable timers on stm32mp135f-dk a7-examples

Enable timer examples that are normally kept disabled by default.
TIM14_CH1 and TIM3_CH4 share the same pin with UART8_RX and DFSDM1_DATIN1
respectively, on RPI connector. TIM14/UART8 and TIM3/DFSDM1 are
mutually exclusive, so keep them disabled by default here.

Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Change-Id: I5659a86e66fc29442b9a443a79180e01c1aa5d2f
---
 arch/arm/boot/dts/st/stm32mp135f-dk-a7-examples.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/st/stm32mp135f-dk-a7-examples.dts b/arch/arm/boot/dts/st/stm32mp135f-dk-a7-examples.dts
index 02dd01732705..d3dbbabe800c 100644
--- a/arch/arm/boot/dts/st/stm32mp135f-dk-a7-examples.dts
+++ b/arch/arm/boot/dts/st/stm32mp135f-dk-a7-examples.dts
@@ -13,6 +13,14 @@ / {
 	compatible = "st,stm32mp135f-dk-a7-examples", "st,stm32mp135f-dk", "st,stm32mp135";
 };
 
+&timers4 {
+	status = "okay";
+};
+
+&timers8 {
+	status = "okay";
+};
+
 &uart8 {
 	status = "okay";
 };
-- 
2.39.5

